CI compilation failed: cannot find value `default_nullability` in this scope

### Describe the bug

There's a merge conflict with #10640.

### To Reproduce

```sh
error[E0425]: cannot find value `default_nullability` in this scope
    --> datafusion/substrait/src/logical_plan/producer.rs:1414:38
     |
1414 |                         nullability: default_nullability,
     |                                      ^^^^^^^^^^^^^^^^^^^ not found in this scope
```

### Expected behavior

_No response_

### Additional context

_No response_

--- a/datafusion/substrait/src/logical_plan/producer.rs
+++ b/datafusion/substrait/src/logical_plan/producer.rs
@@ -1411,7 +1411,7 @@ fn to_substrait_type(dt: &DataType, nullable: bool) -> Result<substrait::proto::
                 parameter: Some(parameter::Parameter::DataType(substrait::proto::Type {
                     kind: Some(r#type::Kind::I32(r#type::I32 {
                         type_variation_reference: DEFAULT_TYPE_REF,
-                        nullability: default_nullability,
+                        nullability: r#type::Nullability::Unspecified as i32,
                     })),
                 })),
             };
@@ -1419,7 +1419,7 @@ fn to_substrait_type(dt: &DataType, nullable: bool) -> Result<substrait::proto::
                 parameter: Some(parameter::Parameter::DataType(substrait::proto::Type {
                     kind: Some(r#type::Kind::I64(r#type::I64 {
                         type_variation_reference: DEFAULT_TYPE_REF,
-                        nullability: default_nullability,
+                        nullability: r#type::Nullability::Unspecified as i32,
                     })),
                 })),
             };
@@ -1443,7 +1443,7 @@ fn to_substrait_type(dt: &DataType, nullable: bool) -> Result<substrait::proto::
                 kind: Some(r#type::Kind::UserDefined(r#type::UserDefined {
                     type_reference,
                     type_variation_reference: DEFAULT_TYPE_REF,
-                    nullability: default_nullability,
+                    nullability,
                     type_parameters,
                 })),
             })

